src/types/types.ts

```diff
@@ -1777,7 +1777,7 @@ export interface NftFilter {
   /** The contract address of the NFT. */
   contractAddress: string;
   /** The token id of the NFT. */
-  tokenId: BigNumberish;
+  tokenId?: BigNumberish;
```

Comment:
Please add a comment that omitting this field results in tracking all activity for the provided contract address. Thanks!